packages/mui/src/components/pages/login/index.tsx

```diff
@@ -74,26 +73,48 @@ export const LoginPage: React.FC = () => {
                         <Card>
                             <CardContent>
                                 <Typography
+                                    fontFamily={typography.fontFamily}
                                     component="h1"
-                                    variant="h5"
+                                    variant="h4"
                                     align="center"
-                                    color="#626262"
+                                    sx={{
+                                        fontWeight: "700",
+                                        fontSize: "30px",
+                                        letterSpacing: "-0.04px",
+                                        lineHeight: "38px",
+                                        color: palette.text.secondary,
+                                    }}
```

Comment:
```suggestion
                                    sx={{
                                        fontWeight: "700",
                                        fontSize: "30px",
                                        letterSpacing: "-0.04px",
                                        lineHeight: "38px",
                                        color: text.secondary,
                                    }}
```

You don't have to use `useTheme` for getting the palette. You can see details in this [link](https://mui.com/system/basics/#shorthands)